website/webpack/prismjs-loader.js

```diff
@@ -0,0 +1,26 @@
+// modified version of https://github.com/valor-software/prismjs-loader
+'use strict';
+const Prism = require('prismjs');
+
+module.exports = function loader(content) {
+  const query = this.query
```

Comment:
Moving the configuration to the Webpack config broke the `prismjs-loader`. I will probably look into making a PR for this (I think it is a simple fix, but I'll have to run some tests), but I included the updated source here so that we don't have to wait for that to happen.